core/ArchiveProcessor/Loader.php

```diff
@@ -184,13 +184,18 @@ protected function prepareCoreMetricsArchive($visits, $visitsConverted)
 
             $this->params->setRequestedPlugin('VisitsSummary');
             $this->params->setArchiveOnlyReport(null);
+            $this->params->setIsPartialArchive(false);
 
-            $pluginsArchiver = new PluginsArchiver($this->params);
-            $metrics = $pluginsArchiver->callAggregateCoreMetrics();
-            $pluginsArchiver->finalizeArchive();
+            $metrics = Context::executeWithQueryParameters(['requestedReport' => ''], function () {
+                $pluginsArchiver = new PluginsArchiver($this->params);
+                $metrics = $pluginsArchiver->callAggregateCoreMetrics();
+                $pluginsArchiver->finalizeArchive();
+                return $metrics;
+            });
```

Comment:
Unset `requestedReport` so it's not picked up if we recurse here. If we're getting core metrics for a week, and there isn't a day archive already existing, we'll try to create the archive. But w/ requestedReport set, we may try to archive the other plugin as well as VisitsSummary. (Note: this can be seen in the new test if the code is removed.)